css/style.css

```diff
@@ -161,6 +167,35 @@ button:hover {
 	cursor: pointer;
 }
 
+#board-header-controls {
+	display: flex;
+	align-items: center;
+	position: relative;
+	margin-left: auto;
+}
+
+#board-header-controls-menu {
```

Comment:
This container is much larger than the button `icon-more`, which means that if you click somewhere close to the edge, popover will not be triggered.